README.md

```diff
@@ -15,6 +15,7 @@ This repo is just a collection of data files for testing houndigrade.
 | disks/rhel_dnf            | 1         | yes   | has RHEL dnf repo enabled                                                      |
 | disks/rhel_cert           | 1         | yes   | has RHEL product cert in ``/etc/pki/product/``                                 |
 | disks/rhel_cert2          | 1         | yes   | has RHEL product cert in ``/etc/pki/product-default/``                         |
+| disks/rhel_lvm            | lvm       | yes   | has RHEL release file                                                          |
```

Comment:
Putting a comment on this line just to hold the auto-merge for a moment since I can't seem to comment on the attached binary file… 

Since we're basically just using git as binary file storage here, what would you think about gzipping our disk image files in the repo? Your new 11 MB `rhel_lvm` file shrinks to a mere 16 KB with stock gzip settings since it's mostly empty space.

This would require adding an extra (manual?) step  after git pull for whoever wants to use them, but the tradeoff is that it would allow us to keep adding and using "big" images without making the repo grow as quickly.

Reply:
Actually, maybe my concerns about size are overblown. 😅  I forgot that internally git compresses the data for new refs, and it compresses _very well_ in general. 

Comparing a full clone of this repo versus only master versus only this new branch, the internal `.git` directories only vary by ~10 KB.

```
❯ git clone -q -b add-lvm --single-branch git@github.com:cloudigrade/houndigrade-test-data.git houndigrade-test-data-add-lvm
❯ du -ch houndigrade-test-data-add-lvm/.git | grep total
312K	total
```

```
❯ git clone -q -b master --single-branch git@github.com:cloudigrade/houndigrade-test-data.git houndigrade-test-data-master
❯ du -ch houndigrade-test-data-master/.git | grep total
304K	total
```

```
❯ git clone -q git@github.com:cloudigrade/houndigrade-test-data.git houndigrade-test-data
❯ du -ch houndigrade-test-data/.git | grep total
320K	total
```

I'll leave this conversation thread open in case you have any additional thoughts, but otherwise I think I may be satisfied just accepting this as-is.